JumpscaleCore/servers/threebot/ThreebotServer.py

```diff
@@ -62,15 +62,15 @@ def openresty_server(self):
         return self._openresty_server
 
     def bcdb_get(self, name):
+        zdb_admin = j.clients.zdb.client_admin_get()
+        zdb = zdb_admin.namespace_new(name, secret=self.secret)
+
         if j.data.bcdb.exists(name=name):
-            zdb_admin = j.clients.zdb.client_admin_get()
             if not zdb_admin.namespace_exists(name):
```

Comment:
Just figured out this check is redundant, and don't know why destroy() is called here.